Lib/test/test_mmap.py

```diff
@@ -734,6 +734,14 @@ def test_resize_past_pos(self):
         self.assertRaises(ValueError, m.write_byte, 42)
         self.assertRaises(ValueError, m.write, b'abc')
 
+    def test_concat_repeat_exception(self):
+        # A SystemError was raised on two unsupported sequence operations.
```

Comment:
Just a nitpick, but I think we can drop this comment. If future readers want to know more about the test, they will find it via `git blame`.